quark/Evaluator/pyeval.py

```diff
@@ -11,18 +11,19 @@
 
 MAX_REG_COUNT = 40
 TIMESTAMPS = datetime.now().strftime('%Y-%m-%d')
-LOG_FILENAME = f"{TIMESTAMPS}.quark.log"
+LOG_FILENAME = f'{TIMESTAMPS}.quark.log'
```

Comment:
According to our doc about coding style, the strings is delimited by double quotes (“). 
[link](https://quark-engine.readthedocs.io/en/latest/coding_style.html#strings)

Reply:
@sidra-asa thanks for reminding, I have removed the double quotes fixer